app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -334,6 +344,25 @@ def parse_persistent_volume(persistent_volume)
       new_result
     end
 
+    def parse_persistent_volume_claim(claim)
+      new_result = parse_base_item(claim)
+      new_result.merge!(
+        :desired_access_modes => claim.spec.accessModes.try(:join, ','),
+        :volume_name          => claim.spec.volumeName,
+        :phase                => claim.status.phase,
+        :actual_access_modes  => claim.status.accessModes.try(:join, ','),
+        :capacity             => claim.status.capacity.to_h.map { |k, v| "#{k}=#{v}" }.join(','),
```

Comment:
@zakiva @alongoldboim I don't like this as well. We should think of a better solution.

Any `join` here in this function is calling for a better solution.